lib/puppet/reference/configuration.rb

```diff
@@ -22,10 +22,16 @@
 
     # Now print the data about the item.
     val = object.default
-    if name.to_s == "vardir"
-      val = "/opt/puppetlabs/puppet/cache"
-    elsif name.to_s == "confdir"
-      val = "/etc/puppetlabs/puppet"
+    if name.to_s == 'vardir'
+      val = 'Unix/Linux: /opt/puppetlabs/puppet/cache -- Windows: C:\ProgramData\PuppetLabs\puppet\cache -- Non-root user: ~/.puppetlabs/opt/puppet/cache'
```

Comment:
One thing we could do is use `%ALLUSERSPROFILE%` in place of `C:\ProgramData` everywhere.  I think most Windows users understand that just fine.